view/frontend/web/js/bolt-api-driven-checkout.js

```diff
@@ -36,56 +38,216 @@ define([
         magentoCartTimeStamp: null,
         cartBarrier: null,
         hintsBarrier: null,
+        elementBarrier: null,
```

Comment:
What does this name mean?

Reply:
Updated to "ReadyStatus", prev. value in replacejs.phtml it was "Check" (like: checkPromise, resolveCheckPromise, etc.)